swimster-ui/src/components/PoolMap/PoolMap.jsx

```diff
@@ -0,0 +1,30 @@
+import * as React from 'react'
+import { GoogleMap, LoadScript } from '@react-google-maps/api'
+
+const mapContainerStyle = {
+    width: '100%',
+    height: '100%',
+}
+
+const center = {
```

Comment:
where is this? good to comment that in

Reply:
oh, these are the attributes of the map. will comment that